src/java/fr/paris/lutece/plugins/forms/web/admin/FormResponseJspBean.java

```diff
@@ -0,0 +1,579 @@
+            synchronized( lock )
+            {
+            	if ( form.getMaxNumberResponse( ) != 0 )
+                {
+            		Map<Integer, Integer> responsePerFormMap = new HashMap<>( );
+                    int nNumberReponseForm = responsePerFormMap.computeIfAbsent( form.getId( ), FormHome::getNumberOfResponseForms );
+                    if ( nNumberReponseForm >= form.getMaxNumberResponse( ) )
+                    {
+                    	return I18nService.getLocalizedString( FormResponseUtils.MESSAGE_ERROR_NUMBER_MAX_RESPONSE_FORM, getLocale( ) );
+                    }
+                }
+            }
+        }
+        
+        Map<String, Object> model = getModel( );
+        
+        if ( form.isActive( ) )
+        {
+            if ( _breadcrumb == null )
+            {
+                _breadcrumb = SpringContextService.getBean( form.getBreadcrumbName( ) );
+            }
+
+            _formResponseManager = FormResponseUtils.initFormResponseManager( request, form, _formResponseManager, _frontOffice );
+            if ( _formResponseManager.getFormResponse( ).isFromSave( ) )
+            {
+                _currentStep = _formResponseManager.getCurrentStep( );
+                _stepDisplayTree = new StepDisplayTree( _currentStep.getId( ), _formResponseManager.getFormResponse( ) );
+
+                Object [ ] args = {
+                        _formResponseManager.getFormResponse( ).getUpdate( ),
+                };
+
+                model.put( FormsConstants.MARK_INFO, I18nService.getLocalizedString( FormResponseUtils.MESSAGE_LOAD_BACKUP, args, FormResponseUtils.getLocale( request ) ) );
+            }
+
+            if ( _stepDisplayTree == null || _currentStep.getId( ) != _stepDisplayTree.getStep( ).getId( ) )
+            {
+                _stepDisplayTree = new StepDisplayTree( _currentStep.getId( ), _formResponseManager.getFormResponse( ) );
+                _formResponseManager.add( _currentStep );
+            }
+            
+            Map<String, Object> modelForStep = _breadcrumb.getModelForCurrentStep( request, _formResponseManager );
+            modelForStep.put( SecurityTokenService.MARK_TOKEN, SecurityTokenService.getInstance( ).getToken( request, FormResponseUtils.ACTION_SAVE_FORM_RESPONSE ) );
+            _stepDisplayTree.addModel( modelForStep );
+
+            model = FormResponseUtils.getFormStepModel( form, request, model, _breadcrumb, _formResponseManager, _stepDisplayTree, _frontOffice );
+            fillCommons( model );
+            
+            String strTitleForm = I18nService.getLocalizedString( FormResponseUtils.MESSAGE_STEP_TITLE, new String [ ] {
+                    form.getTitle( ), _currentStep.getTitle( )
+            }, getLocale( ) );
+            
+            model.put( MARK_FORM_TITLE, strTitleForm );
+        }
+        else
+        {
+        	return I18nService.getLocalizedString( FormResponseUtils.MESSAGE_ERROR_INACTIVE_FORM, getLocale( ) );

```

Comment:
Have you tested the behavior of this return!!!!?
use redirectView please.
Example:
            addError( FormResponseUtils.MESSAGE_ERROR_NUMBER_MAX_RESPONSE_FORM, getLocale( ) );
return redirectView( request, VIEW_......... );